docs/source/using/basic-usage.adoc

```diff
@@ -301,7 +301,8 @@ Using the proxy options will transparently configure the Docker daemon as well a
 
 [NOTE]
 ====
-- `minishift start` honors the environment variables `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY`.
+- `minishift start` honors the environment variables `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` (both uppercase and lower case).
+The lower case `http_proxy`, `https_proxy`, `no_proxy` take higher precedence than `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY`.
```

Comment:
Could be changed to, `The lower case versions of the variables takes precedence over the uppercase ones.` to avoid spelling out the variable multiple times.

Reply:
'takes precedence over' also sounds better.